### PR TITLE
refactor(tip-manager): tailwindify

### PR DESCRIPTION
**Pull request by julio8a**
Supports #1500

### CODE REVIEW

**Comment by julio8a** [comment]
Not sure why this is showing tooltip, which didn't change.

File: src/components/calcite-tip-manager/calcite-tip-manager.scss [hunk 1]
```diff
@@ -1,50 +1,51 @@
...
-  @include focus-style-base();
   &:focus {
-    @include focus-style-outset();
+    @apply focus-outset;
   }
 }
 
 ::slotted(calcite-tip) {
+  @apply border-none;
   border: none;
```
Comment (eriklharper) [inline src/components/calcite-tip-manager/calcite-tip-manager.scss:49]: ☝️ remove `border: none`

**Review by eriklharper — CHANGES_REQUESTED** [review]

File: src/components/calcite-tip-manager/calcite-tip-manager.scss (hunk 1 above)
Comment (julio8a) [reply src/components/calcite-tip-manager/calcite-tip-manager.scss:49]: Good catch!

**Review by eriklharper — APPROVED** [review]

File: src/components/calcite-tip-manager/calcite-tip-manager.scss [hunk 2]
```diff
@@ -59,35 +59,31 @@
...
-  align-items: center;
-  justify-content: center;
-  padding: var(--calcite-spacing-quarter) 0;
+  @apply flex items-center justify-center py-1 px-0;
 }
 
 .page-position {
-  @include font-size(-4);
-  margin: 0 var(--calcite-spacing-half);
+  @apply text--2h my-0 mx-2;
```
Comment (paulcpederson) [inline src/components/calcite-tip-manager/calcite-tip-manager.scss:66]: I think you replaced `-4` with `-2` here, just confirming that is intentional

**Review by paulcpederson — APPROVED** [review]

File: src/components/calcite-tip-manager/calcite-tip-manager.scss (hunk 2 above)
Comment (julio8a) [reply src/components/calcite-tip-manager/calcite-tip-manager.scss:66]: Yes, this is what is on the Figma design. Thanks for checking. Good eyes Paul, good eyes. You still got it.

**Pull request merged**